src/osvr/ClientKit/EyeTrackerC.cpp

```diff
@@ -31,6 +33,9 @@
 // Standard includes
 // - none
 
-namespace osvr {
-namespace common {} // namespace common
-} // namespace osvr
+
```

Comment:
Just like the header, you don't need this file - there's no need to "free" eye data because it doesn't persist after the callback exits.